samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -9,6 +9,7 @@
  */
 
 using System.Net.Http;
+using Sentry.Protocol;
```

Comment:
Lets move the logger log level to the root namaespace to avoid having to import this?

Reply:
moved both new types, `SentryLog` and `SentryLogLevel`